lib/arith.gd

```diff
@@ -966,8 +966,16 @@ DeclareCategoryCollections("IsZDFRECollection");
 ##  <Filt Name="IsMatrix" Arg='obj' Type='Category'/>
 ##
 ##  <Description>
-##  A <E>matrix</E> is a list of lists of equal length whose entries lie in a
-##  common ring.
+##  By convention <E>matrix</E> is a list of lists of equal length whose
+##  entries lie in a common ring.
+##  <P/>
+##  For technical reasons laid out at the top of Chapter <Ref Chap="Matrices"/>,
+##  the filter <Ref Filt="IsMatrix"/> is a synonym for a table of ring elements,
+##  (see <Ref Filt="IsTable"/> and <Ref Filt="IsRingElement"/>). This means that
+##  <Ref Filt="IsMatrix"/> returns <C>true</C> for tables such as 
+##  <C>[[1,2],[3]]</C>.
+##  If necessary, <Ref Prop="IsRectangularTable"/> can be used to test whether
+##  an object is a matrix manually.
```

Comment:
I think this would be clearer if it read `... and object is a list of lists of equal length manually.`